trax/supervised/configs/bert_pretraining.gin

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 The Trax Authors.
```

Comment:
shouldn't this file be named bert_nsp_pretraining.gin?

Reply:
Actually no, because it uses both NSP and MLM as in original bert, but for clarity i changed the nomenclature a bit. Now i have bert_pretraining.gin (both losses), bert_pretraining_onlymlm.gin and bert_pretraining_onlynsp.gin and I think it should be clear.